.github/workflows/on-push-branch-main.yml

```diff
@@ -39,33 +39,64 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  update-visual-snapshots:
+    name: Update Visual Snapshots
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install AWS CLI
+        run: |
+          curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.22.35.zip" -o "awscliv2.zip"
+          unzip awscliv2.zip
+          sudo ./aws/install --update
+      - name: Configure AWS CLI
+        env:
+          CLOUDFLARE_R2_ACCESS_KEY_ID: ${{ secrets.CLOUDFLARE_R2_ACCESS_KEY_ID }}
+          CLOUDFLARE_R2_SECRET_ACCESS_KEY: ${{ secrets.CLOUDFLARE_R2_SECRET_ACCESS_KEY }}
+        # https://developers.cloudflare.com/r2/examples/aws/aws-cli
+        run: |
+          aws configure set region auto
+          aws configure set output json
+          aws configure set aws_access_key_id $CLOUDFLARE_R2_ACCESS_KEY_ID
+          aws configure set aws_secret_access_key $CLOUDFLARE_R2_SECRET_ACCESS_KEY
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/pnpm
+      - run: npx playwright test --update-snapshots
+      - run: aws s3 sync ./dist/snapshots s3://${{ vars.CLOUDFLARE_R2_BUCKET_NAME }}/snapshots --delete --endpoint-url ${{ vars.CLOUDFLARE_R2_ENDPOINT }}
```

Comment:
Just to make sure I follow, so `--delete` will remove any files in the `/snapshots` bucket that aren't in `./dist/snapshots`, right?

Reply:
Yep!